manifest_indexing/manifest_indexing_job.py

```diff
@@ -9,30 +9,45 @@
 import json
 import logging
 
-from utils import write_csv, download_file, upload_file_to_s3_and_generate_presigned_url
-
 from gen3.tools.indexing import index_object_manifest
 
+from settings import JOB_REQUIRES
+from utils import (
+    write_csv,
+    download_file,
+    upload_file_to_s3_and_generate_presigned_url,
+    check_user_permission,
+)
+
 logging.basicConfig(filename="manifest_indexing.log", level=logging.INFO)
 logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
 
 
 if __name__ == "__main__":
     hostname = os.environ["GEN3_HOSTNAME"]
     input_data = os.environ["INPUT_DATA"]
-
-    input_data_json = json.loads(input_data)
+    access_token = os.environ["ACCESS_TOKEN"]
 
     with open("/manifest-indexing-creds.json") as indexing_creds_file:
```

Comment:
is this file created automatically?

Reply:
It is job config created by sower service. We can configure the content by changing the block in sower config. 
https://github.com/uc-cdis/cdis-manifest/blob/8497b41c621b6e6cbefe43efd941e2e547892a63/nci-crdc-staging.datacommons.io/manifest.json#L43